### PR TITLE
Bump fastapi version

### PR DESCRIPTION
**Pull request by msnwatson**
## What was the problem?

Associated tickets or Slack threads:
- #?

## How to test this PR
- Step 1
- Step 2[^secrel]

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

**Review by chengjie8 — APPROVED** [review]
LG

**Review by lukey-luke — APPROVED** [review]

File: domain-cc/cc-app/src/python_src/pydantic_models.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ class Claim(BaseModel):
     claim_type: str = "claim_for_increase"
     contention_text: Optional[str]  # marked optional to retain compatibility with v1
 
-    @root_validator()
+    @root_validator(pre="before")
```
Comment (lukey-luke) [inline domain-cc/cc-app/src/python_src/pydantic_models.py:13]: Should bump pydantic + fastapi versions, switch to `@model_validator` then, added a ticket to tackle this tech debt [here](https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/374)

**Review by dfitchett — APPROVED** [review]

**Comment by msnwatson** [comment]
Just noting, that failing statuses are happening on develop without these changes and are unrelated. Going ahead with merge for now.

**Comment by raudabaugh** [comment]
Do we need to bump the fastapi version in `domain-ee/ee-max-cfi` as well? @msnwatson @department-of-veterans-affairs/benefits-employee-exp-engineers

**Comment by msnwatson** [comment]
@raudabaugh  aha, i just found this actually https://github.com/department-of-veterans-affairs/abd-vro/pull/2554

**Pull request merged**